<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DynamicChangingCamera</class>
 <widget class="QDialog" name="DynamicChangingCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>199</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>251</width>
    <height>199</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>251</width>
    <height>199</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Camera Change</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #E3EAFF;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="viewingAngleHint">
       <property name="text">
        <string>Viewing angle</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="viewingAngleInputField">
       <property name="styleSheet">
        <string notr="true">background: white;
border-radius: 5px;
height: 30px;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="visibilityRangeHint">
       <property name="text">
        <string>Visibility range</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="visibilityRangeInputField">
       <property name="styleSheet">
        <string notr="true">background: white;
border-radius: 5px;
height: 30px;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="positionXHint">
       <property name="text">
        <string>Position X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="positionXInputField">
       <property name="styleSheet">
        <string notr="true">background: white;
border-radius: 5px;
height: 30px;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="positionYHint">
       <property name="text">
        <string>Position Y</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="positionYInputField">
       <property name="styleSheet">
        <string notr="true">background: white;
border-radius: 5px;
height: 30px;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="rotationAngleHint">
       <property name="text">
        <string>Rotation angle</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="rotationAngleInputField">
       <property name="styleSheet">
        <string notr="true">background: white;
border-radius: 5px;
height: 30px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
background: white;
border-radius: 5px; 
height: 30px;
}
QPushButton:hover
{
background: rgb(244, 244, 244);
}


</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
background: white;
border-radius: 5px; 
height: 30px;
}
QPushButton:hover
{
background: rgb(244, 244, 244);
}


</string>
       </property>
       <property name="text">
        <string>Cancel </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
